File: app/src/main/java/com/braingroom/tutor/model/data/ListDialogData.java
package com.braingroom.tutor.model.data;

import java.util.LinkedHashMap;
import java.util.Map;

/**
 * Created by godara on 13/10/17.
 */

public class ListDialogData {
    private LinkedHashMap<String, Integer> items;

    public ListDialogData(LinkedHashMap<String, Integer> items) {
        this.items = items;
    }

    public Map<String, Integer> getItems() {
        return items;
    }

    public void setItems(LinkedHashMap<String, Integer> items) {
        this.items = items;
    }
}

File: app/src/main/java/com/braingroom/tutor/model/data/ListDialogData.kt
package com.braingroom.tutor.model.data

import java.util.LinkedHashMap

/*
 * Created by godara on 13/10/17.
 */

class ListDialogData(items: LinkedHashMap<String, Int>?) {

    private var items: LinkedHashMap<String, Int>

    init {
        this.items = items ?: LinkedHashMap()
    }

    fun getItems(): LinkedHashMap<String, Int> {
        return items ?: LinkedHashMap()
    }

    fun setItems(items: LinkedHashMap<String, Int>?) {
        this.items = items ?: LinkedHashMap()
    }
}
